docs/CustomFonts.md

```diff
@@ -0,0 +1,48 @@
+# Using Custom Fonts
+
+To use custom fonts in Appcues Experiences that are rendering inside of Ionic applications, those fonts must be registered with the native iOS and Android projects for your app. Please follow the instructions in each section below to register the fonts.
+
```

Comment:
I might include the paragraph from the RN docs about naming fonts?

https://github.com/appcues/appcues-react-native-module/blob/main/docs/ExpoManagedWorkflowCustomFonts.md#naming-font-files